src/game/ZONES.ts

```diff
@@ -64,6 +65,26 @@ const ZONES: Zone[] = [
       },
     ],
   },
+  {
+    id: 33,
+    name: 'Aberrus',
+    frozen: false,
+    encounters: [],
+    brackets: {
+      min: 402,
+      max: 450,
+      bucket: 3,
+      type: 'Item Level',
+    },
```

Comment:
this might have to change to match the wcl brackets. not sure what this does exactly tbh :thinking:

Reply:
It's actually not used anywhere and neither are partitions.

Reply:
We could update the query that's used to use partitions if we so chose - this would only impact the Character Parses page and I don't know how often that's actually used.